src/components/Form.js

```diff
@@ -53,25 +59,25 @@ import {
 	________________________________________________________________
 	Using Component:
```

Comment:
Is there a reason why Example is Always Capitalized?

Reply:
no